src/debugpy/adapter/clients.py

```diff
@@ -143,12 +143,23 @@ def initialize_request(self, request):
         self._initialize_request = request
 
         exception_breakpoint_filters = [
-            {"filter": "raised", "label": "Raised Exceptions", "default": False},
-            {"filter": "uncaught", "label": "Uncaught Exceptions", "default": True},
+            {
+                "filter": "raised",
+                "label": "Raised Exceptions",
+                "default": False,
+                "description": "Break whenever any exception is raised.",
+            },
+            {
+                "filter": "uncaught",
+                "label": "Uncaught Exceptions",
+                "default": True,
+                "description": "Break when the process is exiting due to unhandled exception.",
+            },
             {
                 "filter": "userUnhandled",
                 "label": "User Uncaught Exceptions",
                 "default": False,
+                "description": "Break when exception escapes into library code.",
```

Comment:
@luabud @brettcannon For input on the text being used for various exception breakpoint.

Reply:
```suggestion
                "description": "Break when an exception escapes into library code.",
```

Maybe "3rd-party" instead of "library"? No real opinion honestly.

Reply:
It has always been a tricky one to name concisely. The only baseline that we have here is "just my code" as the name of the related breakpoint feature.  But it's basically "stdlib + site-packages" by default, and there are also ways to customize it. So "third-party" is not necessarily accurate if there are some first-party dependencies that just happen to be installed into site-packages (including venv).

Reply:
"non-user code" maybe? But also no preference here 😅 library code is also fine as is. a

Reply:
"Non-user code" is where I started, but then it's kinda redundant wrt the option name; so I wanted the tooltip to give more clues as to what "non-user" actually means.

The other thing is that "user code" makes sense to *us*, but less so to the actual user - to them, it's "my code"! E.g. consider "just my code" vs "user-code only"... the catch here is that we label the option itself "User uncaught exceptions", so it's already wrong in this sense, tooltip or no tooltip. But we could rename it to "Exceptions not caught in my code" or something like that?